src/Tizen.NUI.Extension/Markup/ViewExtensions.cs

```diff
@@ -644,8 +644,13 @@ public static UIColor Color(this View view)
             {
                 return (UIColor)token;
             }
+
+            var color = view.Color;
+            if (color == null)
+                return UIColor.Transparent;
+
             //FIXME: we need to set UI value type directly without converting reference value.
-            return new UIColor(view.Color);
+            return new UIColor(color);
```

Comment:
How about this?

```C#
return new UIColor(view.Color ?? UIColor.Transparent);
```

Reply:
UIColor do not have UIColor(UIColor) but I reduce the line more simply.